common/app/views/fragments/containers/index.scala.html

```diff
@@ -11,7 +11,7 @@
 @* TODO Consolidate this with the master container template to get rid of repetition *@
 
 @defining(page.webTitle) { case (webTitle) =>
-    <section class="@GetClasses.forTagContainer(webTitle.nonEmpty)"
+    <section class="@GetClasses.forTagContainer(webTitle.nonEmpty, adFree = request.headers.keys.exists(_ equalsIgnoreCase "X-Gu-Commercial-Ad-Free"))"
```

Comment:
would it make sense to an `adFree` value in https://github.com/guardian/frontend/blob/513e751f93f74db344cc7faf66f0c4227a4adf34/common/app/implicits/Requests.scala